src/docs/plugins/upgrade-help.md

```diff
@@ -0,0 +1,44 @@
+---
+eleventyNavigation:
+  key: Upgrade Help
+  order: 1
+  excerpt: A plugin to help you upgrade your Eleventy project from Eleventy v1.0 to v2.0.
+---
+
+# Eleventy Upgrade Help
+
+{% tableofcontents %}
+
```

Comment:
I think that we should include a part to specify for which version to which version this plugin will help you to update. The excerpt includes this info right now, but not the page itself.

Also I think it's valuable to have a section for each major version of 11ty, so a level two heading for Eleventy 2 ->  Eleventy 3 and one for Eleventy 1 -> Eleventy 2 and one for Elewventy 0.X -> Eleventy 1. That way a user can use the plugin step by step to update e.g. from eleventy 1 to 3 and this also allows for changing how to use the plugin for each major version.

As an alternative we could also make this generic over all major versions and just include this line from the original Readme:

> The major version of this plugin will always match the major version of Eleventy that you’re upgrading to.

Reply:
I don't necessarily have the bandwidth to polish this up on my own now that the PR has been open for some time & my plate has filled up a bit more.. Any interest in taking it over @Snapstromegon ? Or anyone else from the community, feel free!